### PR TITLE
Kelseydev

### PR DESCRIPTION
**Pull request by kelseyjewell**

### CODE REVIEW

File: app/src/main/java/rainbow_unicorns/rainbow_unicorns/components/decisions/DecisionsActivity.java [hunk 1]
```diff
@@ -12,4 +12,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_decisions);
     }
+
+    //public void addDecidedRestaurant(Restaurant ){
+
+    //}
```
Comment (hdavidzhu) [inline app/src/main/java/rainbow_unicorns/rainbow_unicorns/components/decisions/DecisionsActivity.java]: You don't have to comment these out, or delete them

File: app/src/main/java/rainbow_unicorns/rainbow_unicorns/models/Restaurant.java [hunk 2]
```diff
@@ -1,4 +1,13 @@
 package rainbow_unicorns.rainbow_unicorns.models;
 
 public class Restaurant {
+    private String name;
+
+    public Restaurant(String temp) {
```
Comment (hdavidzhu) [inline app/src/main/java/rainbow_unicorns/rainbow_unicorns/models/Restaurant.java]: String name

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -2,4 +2,8 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+
+    <dimen name="button_width">40dp</dimen>
+    <dimen name="button_height">40dp</dimen>
```
Comment (hdavidzhu) [inline app/src/main/res/values/dimens.xml]: button_size

**Comment by kelseyjewell** [comment]
am i requesting a pull correctly...?

File: app/src/main/java/rainbow_unicorns/rainbow_unicorns/components/home/HomeActivity.java [hunk 4]
```diff
@@ -1,15 +1,91 @@
...
 
+import butterknife.OnClick;
 import rainbow_unicorns.rainbow_unicorns.R;
+import rainbow_unicorns.rainbow_unicorns.models.Restaurant;
 
 public class HomeActivity extends AppCompatActivity {
+    private ImageView appImageView;
+    private ImageButton imageButton;
+    private Queue<Restaurant> allRest;
+    private Restaurant currRest;
```
Comment (hdavidzhu) [inline app/src/main/java/rainbow_unicorns/rainbow_unicorns/components/home/HomeActivity.java:19]: feel free to spell these out in the future, it reduces mental load

File: app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -2,4 +2,9 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+
+    <dimen name="button_size">40dp</dimen>
+    <dimen name="button_margin">40dp</dimen>
```
Comment (hdavidzhu) [inline app/src/main/res/values/dimens.xml:8]: These could be more specific, rather than `button_size` it could be `btn_choose_size`

**Comment by hdavidzhu** [comment]
LGTM

**Pull request merged**